pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -194,11 +194,12 @@ func (p *DNSProxy) GetRules(endpointID uint16) restore.DNSRules {
 						IPs[ip] = struct{}{}
 						count++
 						if count > 1000 {
+							total := len(p.LookupIPsBySecID(nid))
 							log.WithFields(logrus.Fields{
 								logfields.EndpointID:            endpointID,
 								logfields.Port:                  port,
 								logfields.EndpointLabelSelector: cs,
-							}).Warning("Too many DNS rules or destinations for a port, skipping the rest")
+							}).Warningf("Too many (=%d) DNS rules or destinations for a port, skipping the rest", total)
```

Comment:
Would it make more sense as a log field, rather than a formatted string?

Nit: we can also inline `total` to avoid an extra var by just using `len(...)`

Reply:
Thanks, Chris. Pushed a new version with these two changes.